backupmon.sh

```diff
@@ -1157,6 +1157,15 @@ vconfig () {
                 echo -e "Would you like to send a TEST email from BACKUPMON?"
                 if promptyn "(y/n): "; then
 
+                  cemailQuietArg="-verbose"
+                  cemailCheckArg="-versionCheck"
+                  if [ ! -s "$CEMAIL_LIB_FILE_PATH" ]
+                  then
+                      cemailQuietArg="-verbose"
+                      cemailCheckArg="-versionCheck"
+                  fi
+                  _CheckForCustomEmailLibraryScript_ "$cemailCheckArg" "$cemailQuietArg"
```

Comment:
WRT lines **1160** to **1167**:
Is there any specific reason for having the setup & call to the "`_CheckForCustomEmailLibraryScript_` " function **before** sending a test email? IOW, what was the goal for placing the additional call here?

It should not be necessary at all, and it's rather redundant since it was already done at the start of script execution.

Reply:
It was actually my original intention for this section to be the piece that originally downloads the library, so that we don't take up space on people's routers that don't have amtm email enabled. But I need to refine this piece a bit more.